### DISCUSSION TITLE
Duplicate Device Names

### DISCUSSION THREAD

**Discussion by Lorccan** [maxileith/homebridge-appletv-enhanced#661] [Q&A] [answered]
I have linked two Apple TV's to homebridge using the plugin. (The individual ATVs are then paired to HomeKIt.)  I have One Home Screen enabled.  I am also using Node-RED and there the apps on each TV are not differentiated - so I see 'duplicate name' errors.

Is there any way to edit the app names so that I can either add a prefix or suffix.  Any alternative suggestions welcome!

**Comment by Lorccan** [comment, marked as answer]
Answering my own question - I edited the configuredName for each device in `/var/lib/homebridge/appletv-enhanced/<id>/apps.json`

**Reply by maxileith** [reply]
You can also just edit the app names in the home app :)

**Reply by Lorccan** [reply]
> You can also just edit the app names in the home app :)

I can see that now, but it's kinda buried!
